### PR TITLE
Update node16 to latest version

### PR DESCRIPTION
**Pull request by takost**
Update node16 to latest version - 16.20.1.
Node.js will be updated to 16.20.1 for all platforms except windows `win-arm64`, as this platform doesn't have this release. `win-arm64` will be updated to latest available version 16.20.0

### CODE REVIEW

File: src/Misc/externals.sh [hunk 1]
```diff
@@ -149,8 +151,8 @@ fi
 # Download the external tools only for Windows.
 if [[ "$PACKAGERUNTIME" == "win-arm64" ]]; then
     # todo: replace these with official release when available
-    acquireExternalTool "$UNOFFICIAL_NODE_URL/v${NODE16_VERSION}/$PACKAGERUNTIME/node.exe" node16/bin
-    acquireExternalTool "$UNOFFICIAL_NODE_URL/v${NODE16_VERSION}/$PACKAGERUNTIME/node.lib" node16/bin
+    acquireExternalTool "$UNOFFICIAL_NODE_URL/v${NODE16_UNOFFICIAL_VERSION}/$PACKAGERUNTIME/node.exe" node16/bin
```
Comment (fhammerl) [inline src/Misc/externals.sh:154]: +1 for explicit naming

**Review by fhammerl — APPROVED** [review]
LGTM

**Review by KramNosaj01 — COMMENTED** [review]
src/Misc/contentHash/externals/linux-arm
src/Misc/contentHash/externals/linux-arm64
src/Misc/contentHash/externals/osx-arm64
src/Misc/contentHash/externals/osx-x64
src/Misc/contentHash/externals/win-arm64
src/Misc/contentHash/externals/win-x64
src/Misc/externals.sh
takost/update_latest_node16

**Review by KramNosaj01 — COMMENTED** [review]
takost/update_latest_node16 [](url)

**Pull request merged**